mockbook/src/App.vue

```diff
@@ -5,3 +5,87 @@
 <script setup lang="ts"></script>
 
 <style scoped></style>
+
+
+<style>
```

Comment:
2 style tags seems out of place. Prefer to only use 1 style tag instead:
```suggestion
<style>
```

Reply:
agree if the scoped one is not needed but there will be time in Vue where both are needed, even though I tend to avoid using unscoped style tags inside `*.vue` files to not have to chase their rules when debugging